### ISSUE TITLE
github RSA host key is updated.

### ISSUE THREAD

**Issue by r-plus** [guitarrapc/git-shallow-clone-orb#40]
hi @guitarrapc 

https://github.blog/2023-03-23-we-updated-our-rsa-ssh-host-key/

github.com RSA host key is replaced today.
should update the `echo` line of code to new RSA key.

https://github.com/guitarrapc/git-shallow-clone-orb/blob/39f6ba8c7fe433263e012156e84b8d1e84b8f332/src/commands/checkout.yml#L71

**Comment by guitarrapc** [comment]
Thanks, I didn't realize it!

**Comment by guitarrapc** [comment]
I've released v2.5.0.

https://github.com/guitarrapc/git-shallow-clone-orb/releases/tag/v2.5.0
https://circleci.com/developer/orbs/orb/guitarrapc/git-shallow-clone?version=2.5.0

Please update your orb to following.
```
orbs:
  git-shallow-clone: guitarrapc/git-shallow-clone@2.5.0
```

**Comment by r-plus** [comment]
thanks for quick response :)
